pkg/blobinfocache/internal/prioritize/prioritize_test.go

```diff
@@ -155,9 +155,9 @@ func TestCandidateSortStateSwap(t *testing.T) {
 }
 
 func TestDestructivelyPrioritizeReplacementCandidatesWithMax(t *testing.T) {
-	for _, max := range []int{0, 1, replacementAttempts, 100} {
+	for _, max := range []int{0, 1, replacementAttempts, 100, replacementUnknownLocationAttempts} {
```

Comment:
(This file should, in the final version, have real tests for the unknown-location cases.)

Reply:
Yes I agree. Once we have mututal agreement on code and API changes. I will write new tests for this.